app/src/full/java/io/homeassistant/companion/android/notifications/FirebaseCloudMessagingService.kt

```diff
@@ -0,0 +1,64 @@
+package io.homeassistant.companion.android.notifications
+
+import android.util.Log
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import dagger.hilt.android.AndroidEntryPoint
+import io.homeassistant.companion.android.common.data.authentication.AuthenticationRepository
+import io.homeassistant.companion.android.common.data.authentication.SessionState
+import io.homeassistant.companion.android.common.data.integration.DeviceRegistration
+import io.homeassistant.companion.android.common.data.integration.IntegrationRepository
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class FirebaseCloudMessagingService : FirebaseMessagingService() {
+    companion object {
+        const val TAG = "MessagingService"
```

Comment:
Should we change this so we know the log came from firebase? I noticed `MessagingManager` has the same `TAG` so didnt want to confuse them :)

Reply:
Fixed.